### PR TITLE
Frontlight - Add checkbox use system settings

### PR DESCRIPTION
**Pull request by robert00s**
See: https://github.com/koreader/koreader/issues/5205#issuecomment-526935357

Devices with `hasLightLevelFallback = true` (for now Android) has extra checkbutton `Use system settings`. Default checked.

After check button we have Frontlight widget with read only status.

Also frontlight gestures are disabled.

Checkbutton status is written in global variable `light_fallback`.

### CODE REVIEW

File: frontend/apps/reader/modules/readergesture.lua [hunk 1]
```diff
@@ -1302,13 +1307,29 @@ function ReaderGesture:gestureAction(action, ges)
             self.ui:handleEvent(Event:new("ShowFlDialog"))
         end
     elseif action == "increase_frontlight" then
+        -- when using frontlight system settings
+        if lightFrontlight() then
+            UIManager:show(Notification:new{
+                text = _("Frontlight controlled by system settings."),
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readergesture.lua]: is/will now be?

Also I think 1 s might be a bit short.

Comment (robert00s) [reply frontend/apps/reader/modules/readergesture.lua]: 2 s or more?

Comment (Frenzie) [reply frontend/apps/reader/modules/readergesture.lua]: I'd say 2 or 3.

Comment (robert00s) [reply frontend/apps/reader/modules/readergesture.lua]: 2.5 s :)

**Comment by pazos** [comment]
Awesome job @robert00s !

I will review changes at night and submit a PR on top of your branch.

A few things I noticed:

> Devices with hasLightLevelFallback = true (for now Android) has extra checkbutton Use system settings. Default unchecked.

Default is system settings, at least on Android. No problem with that since it can be retrieved using

```lua
function Device:isUsingSystemSettings()
    return android.getScreenBrightness < 0
end
```

**Comment by robert00s** [comment]
Now the system settings are the default (checkbox checked)

**Comment by pazos** [comment]
Fixes #5205

**Pull request merged**